helm/os-vim-driver/values.yaml

```diff
@@ -44,7 +44,20 @@ app:
       messaging:
         # Kafka connection url
         connection_address: foundation-kafka:9092
-  
+
+    security:
+      ssl:
+        enabled: True
+        secret:
+          ## Name of the secret containing the SSL certificate for the non-host based access (the CN of the certificate should match commonName)
+          name: ovd-tls
+          ## If True, the Helm chart installation will generate a new SSL key with a self-signed certificate
+          generate: True
+          ## The Common Name used for the SSL certificate
+          ## The default value here is acceptable but will not produce a completely valid certificate
+          ## You should update this value to the IP address used to access the API on this rule
```

Comment:
Is this true? This value is correct as long as onboard the driver with the internal address "os-vim-driver" which is the main use case?

Reply:
ok